src/views/CharacterListView.js

```diff
@@ -3,33 +3,50 @@ import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
 // import actions
+import { getChars } from "../actions";
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
+	componentDidMount() {
+		// call our action
+		this.props.getChars("https://swapi.co/api/people");
+	}
 
-  componentDidMount() {
-    // call our action
-  }
-
-  render() {
-    if (this.props.fetching) {
-      // return something here to indicate that you are fetching data
-    }
-    return (
-      <div className="CharactersList_wrapper">
-        <CharacterList characters={this.props.characters} />
-      </div>
-    );
-  }
+	render() {
+		console.log(this.props);
+		if (this.props.fetching) {
+			// return something here to indicate that you are fetching data
+			return <div>Retrieving Characters...</div>;
+		}
+		return (
```

Comment:
```suggestion
else if (this.props.error) {
      return <div>{this.props.error.message}</div>;
    }
		return (
```
I would add in some sort of error handling here.